wherehows-web/app/services/search.ts

```diff
@@ -20,7 +20,7 @@ export default class Search extends Service {
       keyword = encode(keyword);
 
       return applicationRoute.transitionTo('search', {
```

Comment:
Why do we automatically transition to the search route just because the user has typed in something for the typeahead? It actually really annoys me as a user of wherehows in this happens vs transitioning when i hit enter or something.

Reply:
Solved